File: challenges/3-area-trapezio/php/jcesar/trapezio.php
<?php
if (sizeof($argv) < 4)
{
    echo 'FALTAM ARGUMENTOS';
    exit(); 
}

$baseMaior = $argv[1];
$baseMenor = $argv[2];
$altura = $argv[3]; 


if (!is_numeric($baseMaior) OR !is_numeric($baseMenor) OR !is_numeric($altura))
{
    echo 'INSIRA 3 NÚMEROS POR FAVOR';
    exit();
}


$area = ($baseMaior + $baseMenor) * $altura / 2;

echo "A área do trapézio é: {$area}";

File: challenges/3-area-trapezio/java/disousadev/Trapezio.java
import java.util.InputMismatchException;
import java.util.Scanner;

public class Trapezio {

    public static void main(String[] args) {

        Scanner scan = new Scanner(System.in);

        try {

            double baseMaior = scan.nextDouble();
            double baseMenor = scan.nextDouble();
            double altura = scan.nextDouble();

            double area = (baseMaior + baseMenor) * altura / 2;

            System.out.println("A área do trapézio é: " + area);

        } catch (InputMismatchException e) {
            System.out.println("Digite apenas números.");
        } finally {
            scan.close();
        }
    }
}